scripts/run-integration-tests.sh

```diff
@@ -38,11 +38,11 @@ main() {
     #Check if Fabric is in the gopath. Fabric needs to be in the gopath for the integration tests
     if [ ! -d "${FABRIC_DIR}" ]; then
         echo "Downloading Fabric Branch v1.4.0"
-        git clone https://github.com/hyperledger/fabric $FABRIC_DIR --branch v1.4.0 --single-branch --depth 1
+        git clone https://github.com/hyperledger/fabric $FABRIC_DIR --branch release-1.4 --single-branch --depth 1
     else
-        FABRIC_VERSION=$(git -C ${FABRIC_DIR} describe --abbrev=0)
-        if [[ ${FABRIC_VERSION} != "v1.4.0" ]]; then
-          echo "Please switch Fabric Repository to tag v1.4.0 before running these tests"
+        FABRIC_VERSION=$(git -C ${FABRIC_DIR} rev-parse --abbrev-ref HEAD)
+        if [[ ${FABRIC_VERSION} != "release-1.4" ]]; then
+          echo "Please switch Fabric Repository to release v1.4.0 before running these tests, it currently is on ${FABRIC_VERSION}"
```

Comment:
Does this have to match exactly the branch name?

If so, the echo Error message should update to say "branch relaese-1.4".